transmart-core-db/src/main/groovy/org/transmartproject/db/accesscontrol/AccessControlChecks.groovy

```diff
@@ -377,4 +369,17 @@ class AccessControlChecks {
         res
     }
 
+    /**
+     * Groovy supports multiple dispatch. So other {@see canPerform} method has to be called.
+     * This class does not have to be @CompileStatic in order for this mechanism to work.
```

Comment:
Would be great, though, to have as much `@CompileStatic` as possible in the code (although groovy has some weird behaviour with inheritance).